src/components/DashboardMedia/DashboardMediaData.js

```diff
@@ -39,6 +40,13 @@ function DashboardMediaData() {
       href: DOCUMENT_PATH,
       iconColor: 'jade-800',
     },
+    {
+      icon: 'Images',
+      name: 'Shared Images',
```

Comment:
[very minor] all our other tiles with multiple words in the dashboard only capitalize the first word, should be "Shared images"

Reply:
Done. Thank you!